BGL/test/BGL/test_cgal_bgl_named_params.cpp

```diff
@@ -46,7 +46,7 @@ template<class NamedParameters>
 void test_no_copyable(const NamedParameters& np)
 {
   typedef typename inp::Get_param<typename NamedParameters::base,inp::visitor_t>::type NP_type;
-  static_assert( boost::is_same<NP_type,std::reference_wrapper<const B> > ::value );
```

Comment:
## Note

See https://en.cppreference.com/w/cpp/language/static_assert:
 - `static_assert(bool, message)` is C++11,
 - `static_assert(bool)` without the message is C++17.